bedrock/settings/base.py

```diff
@@ -1498,7 +1499,7 @@ def get_default_gateway_linux():
 CSP_REPORT_ONLY = config('CSP_REPORT_ONLY', default='false', parser=bool)
 CSP_REPORT_ENABLE = config('CSP_REPORT_ENABLE', default='false', parser=bool)
 if CSP_REPORT_ENABLE:
-    CSP_REPORT_URI = config('CSP_REPORT_URI', default='/csp-violation-capture')
+    CSP_REPORT_URI = config('CSP_REPORT_URI', default='')
```

Comment:
I'd remove default value here, to prevent it from starting if no URI has been defined by `CSP_REPORT_ENABLE` is `True`

Reply:
I decided to remove the `CSP_REPORT_ENABLE` setting entirely. It's no longer needed since we don't have a default URL for the reports. I tested having `None` as the default for the setting and it is disabled as expected.